games/src/main/scala/com/gu/game_review_parser/parsers/StandardParser.scala

```diff
@@ -0,0 +1,117 @@
+package com.gu.game_review_parser.parsers
+
+import com.gu.contentapi.client.model.v1.Content
+import com.gu.game_review_parser.ParsedGameReview
+import org.jsoup.Jsoup
+
+import scala.util.Try
+
+object StandardParser {
+  def parseContent(content: Content): Option[ParsedGameReview] = {
+    val parsed = for {
+      title <- getTitle(content.webTitle)
+      fields <- content.fields
+      body <- fields.body
+      details <- getDetails(body)
```

Comment:
What I mean, is if for example getDetails returns None then reviewer, starRating and reviewSnippet will also be None even if they are in fact present! Separating them out into separate for comprehensions prevents this.